### PR TITLE
Add support for edge beta and canary

### PR DESCRIPTION
**Pull request by nmcc1212**

### CODE REVIEW

File: src/config/activities.ts [hunk 1]
```diff
@@ -20,6 +20,8 @@ import yandex from '@browser-logos/yandex/yandex_256x256.png'
 import copyToClipboard from './copy-to-clipboard.png'
 import min from './min_256x256.png'
 import polypane from './polypane_256x256.png'
+import edgeCan from '@browser-logos/edge-canary/edge-canary_256x256.png'
```
Comment (nmcc1212) [inline src/config/activities.ts]: Browser Logos

File: src/config/activities.ts [hunk 2]
```diff
@@ -141,6 +145,20 @@ export const activities: Activities = {
     hotKey: 'e',
     logo: edgeDev,
   },
+  'Microsoft Edge Canary':{
```
Comment (nmcc1212) [inline src/config/activities.ts]: Added both beta and canary builds

File: src/config/activities.ts [hunk 3]
```diff
@@ -44,6 +46,8 @@ export const activityNames = [
   'Iridium',
   'Maxthon',
   'Microsoft Edge Dev',
+  'Microsoft Edge Canary'
```
Comment (nmcc1212) [inline src/config/activities.ts]: Added to browser list

**Comment by will-stone** [comment]
Thanks. I'm off on hols this weekend but I'll take a look next week when I'm back. Cheers.

**Comment by will-stone** [comment]
Could you please add the logos to the package.json?

**Comment by will-stone** [comment]
Hi. I know yours was first but #141 is closer to completion and covers this browser. Thank you for your contribution though, I really appreciate it 😄

**Pull request closed without merging**